app/src/main/java/org/wikipedia/page/LinkHandler.kt

```diff
@@ -31,6 +32,12 @@ abstract class LinkHandler(protected val context: Context) : JSEventListener, Ur
 
     override fun onUrlClick(url: String, titleString: String?, linkText: String) {
         var href = url
+        if (href.startsWith("mailto:")) {
+            val emailAddress = href.removePrefix("mailto:")
+            FeedbackUtil.composeFeedbackEmail(context, email = emailAddress, subject = "", body = "")
```

Comment:
We can make the `subject`'s default value to `=""`, and then we can skip assigning the empty string for both `subject` and `body`.

Reply:
Done.